align/main.py

```diff
@@ -65,3 +66,24 @@ def schematic2layout(netlist_dir, pdk_dir, netlist_file=None, subckt=None, worki
                     mp.write(f'{true_stem} {true_stem}.gds\n')
                 elif file_.suffix == '.lef' and file_.stem != subckt:
                     lp.write(file_.read_text())
+        # Copy over necessary collateral & run PNR tool
```

Comment:
Should use file mode "wt" instead of just "w" a few lines above.

Reply:
Thanks for catching that. I believe text mode is default since python3 but better to be explicit than implicit.